python/pyspark/pandas/tests/test_dataframe.py

```diff
@@ -1954,6 +1954,41 @@ def test_isin(self):
         with self.assertRaisesRegex(TypeError, msg):
             psdf.isin(1)
 
+        pdf = pd.DataFrame(
+            {
+                "a": [4, 2, 3, 4, 8, 6],
+                "b": [1, None, 9, 4, None, 4],
+                "c": [None, 5, None, 3, 2, 1],
+            },
+        )
+        psdf = ps.from_pandas(pdf)
+
+        if LooseVersion(pd.__version__) >= LooseVersion("1.2"):
+            self.assert_eq(psdf.isin([4, 3, 1, 1, None]), pdf.isin([4, 3, 1, 1, None]))
+        else:
+            expected = pd.DataFrame(
+                {
+                    "a": [True, False, True, True, False, False],
+                    "b": [True, True, False, True, True, True],
+                    "c": [True, False, True, True, False, True],
```

Comment:
This fails with pandas <1.2.

Should be:

```suggestion
                    "a": [True, False, True, True, False, False],
                    "b": [True, False, False, True, False, True],
                    "c": [False, False, False, True, False, True],
```
?

Reply:
Thank you, updated. Sorry about my again mistake!

Reply:
No worries! Thanks for the fix!